apps/web/pages/[user].tsx

```diff
@@ -78,25 +78,23 @@ export function UserPage(props: UserPageProps) {
             isEmbed ? "border-booker border-booker-width  bg-default rounded-md border" : "",
             "max-w-3xl px-4 py-24"
           )}>
-          {isSingleUser && ( // When we deal with a single user, not dynamic group
```

Comment:
isSingleUser is no longer possible because dynamic events are now redirected to [type]